scripts/initialize-citus.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -e
+
+DATABASE=commcarehq_citus
+
+function exec_sql() {
+    HOST=$1
+    DB=$2
+    SQL=$3
+    docker exec -it $HOST psql -U postgres $DB -c "$SQL"
+}
+
+function get_worker_status() {
+    echo "`docker inspect -f {{.State.Health.Status}} citus_worker_1`"
+}
+
+COMPOSE_PROJECT_NAME=citus docker-compose -f docker/citus-compose.yml up -d
+
+MAX_ATTEMPTS=10
+ATTEMPTS=0
+until [[ "`get_worker_status`" == "healthy" || $ATTEMPTS -eq $MAX_ATTEMPTS ]]; do
+    echo "Waiting for worker. Attempt $((ATTEMPTS++))"
+    sleep 1;
+done;
+
+if [ "`get_worker_status`" != "healthy" ]; then
+    echo "Citus worker not healthy: `get_worker_status`"
+    echo $ATTEMPTS
+    exit 1
+fi
+
+echo "## Creating database: $DATABASE"
+exec_sql citus_worker_1 postgres "create database commcarehq_citus;"
+exec_sql citus_master postgres "create database commcarehq_citus;"
```

Comment:
Could these be done as part of a django migration?.

Reply:
no since you need to create the DB before Django can connection. Django does do this for tests but since we need to do some extra work like adding extensions and configuring the worker nodes it seemed reasonable to do it outside of django. Normally this will be handled by our ansible setup.